python/postgresql/yellowbrick/README.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Copyright 2025 Columnar Technologies Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Connecting Python and Yellowbrick with ADBC
+
+## Instructions
+
+> [!TIP]
+> If you already have a Yellowbrick instance running, skip the steps to set up and clean up Yellowbrick.
+
+### Prerequisites
+
+1. [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
+
+2. [Install dbc](https://docs.columnar.tech/dbc/getting_started/installation/)
+
+### Set up Yellowbrick
+
+1. [Install Docker](https://docs.docker.com/get-started/get-docker/)
+
+2. Start a Yellowbrick Community Edition instance:
+
+    ```sh
+    docker run -it --privileged --name yellowbrick -p 443:443 -p 5432:5432 yellowbrickdata/yb-community-edition:latest
```

Comment:
This didn't run for me on macOS. I get:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
WARNING: CPU 'VirtualApple' may not be supported. The system may not come up.
[ERROR] Command failed: ybstart pg --datadir /mnt/ybdata/pgdata -c docker
Timed out waiting for PG start flag.
```

@esadek were you able to get it running?

Reply:
Are you using Docker Desktop? OrbStack?

Reply:
OrbStack